tools/devtool

```diff
@@ -369,6 +369,7 @@ run_devctr() {
     docker run "${docker_args[@]}" \
         --rm \
         --volume /dev:/dev \
+        -v /etc/localtime:/etc/localtime \
```

Comment:
I think that if we install `tzdata` in the contaiener we won't need to mount the `localtime` file.

Reply:
Indeed. However, why do you think installing the package would be a better option?

Reply:
The host may not necessarily have `/etc/localtime`.

Reply:
makes sense

Reply:
done